.eslintrc-base.json

```diff
@@ -61,7 +61,7 @@
     "jsx-a11y/no-noninteractive-element-interactions": "error",
     "jsx-a11y/no-noninteractive-element-to-interactive-role": "error",
     "jsx-a11y/no-noninteractive-tabindex": "error",
-    "jsx-a11y/no-onchange": "error",
+    "jsx-a11y/no-onchange": 0,
```

Comment:
Once I got eslint to work, there is an error in the Footer component, where we have an `onChange` event in `<select>` element.

This rule is outdated as it was based on the behaviors of old browsers. The rule will be deprecated in the next release of `jsx-a11y`: https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/issues/398#issuecomment-729139609

I disabled the rule to prevent false positive error.